spec/fixtures/vcr_cassettes/Stripe-v10.13.0/Spree_CreditCardsController/using_VCR/_new_from_token/when_the_request_to_store_the_customer/card_with_Stripe_is_successful/saves_the_card_locally.yml

```diff
@@ -8,9 +8,9 @@ http_interactions:
       string: card[number]=4242424242424242&card[exp_month]=9&card[exp_year]=2024&card[cvc]=314
     headers:
       User-Agent:
-      - Stripe/v1 RubyBindings/10.12.0
+      - Stripe/v1 RubyBindings/10.13.0
       Authorization:
-      - Bearer <HIDDEN-STRIPE_INSTANCE_SECRET_KEY>
+      - "<HIDDEN-AUTHORIZATION-HEADER>"
```

Comment:
That's interesting. 

This is due to update here: https://github.com/openfoodfoundation/openfoodnetwork/commit/2e101c5fe63a3db899fe25c902fad0a8d7baff35#diff-ce0df5019dcf4bab56f483cde00b919c209c37aed3e8dbf85f13edc37ff032c0R31-R33

We now mask any `Authorization` headers, OR the exact instance secret key. Seems like a good idea to cover from both angles.